applications/GeoMechanicsApplication/custom_elements/geo_steady_state_Pw_piping_element.h

```diff
@@ -32,12 +33,14 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoSteadyStatePwPipingElement : publ
 
     explicit GeoSteadyStatePwPipingElement(IndexType NewId = 0) : Element(NewId) {}
 
-    GeoSteadyStatePwPipingElement(IndexType NewId, GeometryType::Pointer pGeometry)
+    GeoSteadyStatePwPipingElement(IndexType NewId, const GeometryType::Pointer& pGeometry)
```

Comment:
I'm not sure how to comply with the Kratos Style Guide here. Perhaps:
```suggestion
    GeoSteadyStatePwPipingElement(IndexType NewId, const GeometryType::Pointer& rpGeometry)
```
???

Reply:
took the suggestion.